.github/workflows/messaging.yml

```diff
@@ -0,0 +1,71 @@
+name: messaging
+
+on:
+  pull_request:
+    paths:
+    - 'Firebase/Messaging/**'
+
+    #Podspec
+    - 'FirebaseMessaging.podspec'
+
+    - '.github/workflows/messaging.yml'
+  schedule:
+    # Run every day at 11pm (PST) - cron uses UTC times
+    - cron:  '0 7 * * *'
+
+jobs:
+  messaging:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup Bundler
+      run: scripts/setup_bundler.sh
+    - name: Install Secret GoogleService-Info.plist
+      env:
+         plist_secret: ${{ secrets.StoragePlistSecret }}
+      run: ./scripts/decrypt_gha_secret.sh scripts/gha-encrypted/storage-db-plist.gpg \
+          FirebaseStorage/Tests/Integration/Resources/GoogleService-Info.plist "$plist_secret"
+    - name: BuildAndTest # can be replaced with pod lib lint with CocoaPods 1.10
+      run: scripts/third_party/travis/retry.sh scripts/build.sh Storage all
+
+  pod_lib_lint:
```

Comment:
This name is spelled inconsistently with `messaging-cron-only` (i.e. snake_case vs kebab-case). Rename these to `pod-lib-lint`?

Also, if you're up for it, fix core.yml as well which also has this inconsistency.

Reply:
I think it's because core also already has a 'pod-lib-lint'. I think the difference is one with cron job running pod lib lint?

Reply:
Oh you meant the format. Got it.